src/ReaderScreen/utils/gutkahtml.js

```diff
@@ -80,6 +80,24 @@ const htmlTemplate = (backColor, fontFace, content, theme, savePosition) => `<!D
     .right{
       text-align:right
     }
+    /* Minimal scrollbar */
+    #sb {
+      position: fixed;
+      right: 0;
+      top: 0;
+      width: 6px;
+      height: 100vh;
+      z-index: 99999;
+      pointer-events: none;
+    }
+    #sb-t {
+      position: absolute;
+      right: 1;
```

Comment:
This should either be 0, or 1px, right? Is there a reason for it being 1 without the unit?

Reply:
fixed.